cortex/test_cli.py

```
@@ -0,0 +1,206 @@
+    def test_get_api_key_openai(self):
+        api_key = self.cli._get_api_key()
+        self.assertEqual(api_key, 'test-key')
+    
+    @patch.dict(os.environ, {'ANTHROPIC_API_KEY': 'test-claude-key', 'OPENAI_API_KEY': ''}, clear=True)
+    def test_get_api_key_claude(self):
+        api_key = self.cli._get_api_key()
+        self.assertEqual(api_key, 'test-claude-key')
+    
+    @patch.dict(os.environ, {}, clear=True)
+    @patch('sys.stderr')
+    def test_get_api_key_not_found(self, mock_stderr):
```

Ruff (0.14.4)

Unused method argument: `mock_stderr`

(ARG002)